### DISCUSSION TITLE
Customization of Cards layout - Search Results Box

### DISCUSSION THREAD

**Discussion by piranhap** [microsoft-search/pnp-modern-search#3054] [Q&A] [answered]
Hello everyone! 
I'm setting up a KB-website in SPO. There is a page that contains a search box and a search results box to display all the available KB articles (Site pages .aspx) 
When looking for terms the search result box displays looking for content, title, keywords, etc. The problem is that with the card layout (compact version) you have to click on the title to go to the page, not the box. I'm wondering if there is a way to tweak that. 
Images for more reference:
https://i.imgur.com/fSnBOOG.png
https://i.imgur.com/nVgn56I.png

**Comment by wobba** [comment]
* Pick the card layout
* Click the `{}` button next to "edit result template. This will copy the default card layout and allow you to edit it
* Add an `<a href`> tag around the card component

```html
{{#> resultTypes item=item}}
                <a href="https://www.contoso.com">
                    <pnp-documentcard class="template--card" 
                        data-ui-test-id="resultCard" 
                        data-item="{{JSONstringify item}}" 
                        data-fields-configuration="{{JSONstringify @root.properties.layoutProperties.documentCardFields}}" 
                        data-enable-preview="{{@root.properties.layoutProperties.enablePreview}}" 
                        data-show-file-icon="{{@root.properties.layoutProperties.showFileIcon}}" 
                        data-is-compact="{{@root.properties.layoutProperties.isCompact}}" 
                        data-context="{{JSONstringify @root}}"
                        data-instance-id="{{@root.instanceId}}"
                        data-theme-variant="{{JSONstringify @root.theme}}"
                        data-allow-item-selection="{{@root.properties.itemSelectionProps.allowItemSelection}}"
                        data-item-key="{{@root.paging.currentPageNumber}}{{@index}}"                     
                        data-selected-keys="{{JSONstringify @root.selectedKeys}}"
                        data-index="{{@index}}"
                        >
                    </pnp-documentcard>
                </a>
                {{/resultTypes}}
```

**Reply by piranhap** [reply]
Thank you for your reply. I see that will make you go to the contoso website (for example) how can I set it up, so it sends people to the site they are clicking.

**Reply by piranhap** [reply]
I figured out by referencing {{Item.Path}}
the problem is that messed up the allignment 

https://i.imgur.com/eqBSRjk.png

any ideas @wobba ? Thank you very much!

**Reply by wobba** [reply, marked as answer]
Put some width css around it maybe. I see the middle item is wider. You might be better off not using cards and using html/css for your entire view instead.
